migrations/2025-04-07-141445_disable-email-notifications/up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE local_site
+    ADD COLUMN disable_email_notifications bool NOT NULL DEFAULT FALSE;
```

Comment:
Doesn't matter to me, but `local_user` uses a positive `send_notifications_to_email`. 

At some point we should probably go through the API and decide on what basis we choose to say `hide` vs `show` also.

No need to change this one of course.

Reply:
Lemmy sends out email notifications by default, so it is something that needs to be disabled explicitly. Thats why disable makes more sense for me.